webapp/src/views/SignUp.vue

```diff
@@ -157,9 +157,10 @@
 
 <script>
   import axios from 'axios';
-  import {LOCAL_PUBLIC_SUFFIXES} from '../env';
   import {domain_pattern, email_pattern} from '../validation';
 
+  let LOCAL_PUBLIC_SUFFIXES = process.env.VUE_APP_LOCAL_PUBLIC_SUFFIXES.split(' ');
```

Comment:
`const`

Also, we don't define this as a file-scoped variable in other contexts (but fine with me, I guess you have a good reason)

Reply:
yeah, `this` is unavailable in the pattern checking mechanism further down the file